libs/execution_context/ebpf_maps.c

```diff
@@ -59,6 +46,26 @@ typedef struct _ebpf_core_lpm_map
     uint8_t data[1];
 } ebpf_core_lpm_map_t;
 
+typedef struct _ebpf_core_ring_buffer_map
+{
+    ebpf_core_map_t core_map;
+    // Flag that is set the first time an async operation is queued to the map.
+    // This flag only transitions from off -> on. When this flag is set,
+    // updates to the map acquire the lock and check the async_contexts list.
+    // Note that queueing an async operation thus causes a perf degradation
+    // for all subsequent updates, so should only be allowed to admin.
+    bool async_contexts_trip_wire;
```

Comment:
Two issues:
1) We don't need a list of async contexts.
2) Trip wire doesn't make sense as we don't use it anymore.

Reply:
Will be addressed in #669